src/mam4xx/tropopause.hpp

```diff
@@ -1,22 +1,19 @@
 #ifndef MAM4XX_TROPOPAUSE_HPP
 #define MAM4XX_TROPOPAUSE_HPP
 
-#include <haero/math.hpp>
-#include <mam4xx/aero_config.hpp>
-
 #include <fstream>
```

Comment:
Are `fstream` and `iostream` needed? I haven't looked closely, but I don't notice any calls to C++ streams here.

Reply:
I do not think so. I will remove it.